pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -514,7 +514,9 @@ def run(
             }
             if self.config.eta > 0 and self.initial_state != qutip.tensor(
                 [
-                    self._hamiltonian.basis["g"]
+                    self._hamiltonian.basis[
+                        "u" if self._hamiltonian._interaction == "XY" else "g"
+                    ]
```

Comment:
This might qualify this change as an hotfix, don't you think?

Reply:
I think so as well :)